README.md

```diff
@@ -2,37 +2,41 @@
 
 This module handles opiniated VPC Service Controls and Access Context Manager configuration and deployments.
 
+
+## Compatibility
+This module is meant for use with Terraform 0.12. If you haven't [upgraded][terraform-0.12-upgrade] and need a Terraform 0.11.x-compatible version of this module, the last released version intended for Terraform 0.11.x
+is [0.1.0](https://github.com/terraform-google-modules/terraform-google-vpc-service-controls/releases/tag/v0.1.0).
+
 ## Usage
 The root module only handles the configuration of the [access_context_manager_policy resource](https://www.terraform.io/docs/providers/google/r/access_context_manager_access_policy.html). For examples on how to use the root module with along with other submodules to configure all of VPC Service Controls and Access Context Manager resources, see the [examples](./examples/) folder and the [modules](./modules/) folder
 
 ```hcl
 provider "google" {
   version     = "~> 2.5.0"
-  credentials = "${file("${var.credentials_path}")}"
 }
 
 module "org_policy" {
   source      = "terraform-google-modules/vpc-service-controls/google"
-  parent_id   = "${var.parent_id}"
-  policy_name = "${var.policy_name}"
+  parent_id   = var.parent_id
+  policy_name = var.policy_name
 }
 
 module "access_level_members" {
   source   = "terraform-google-modules/vpc-service-controls/google//modules/access_level"
-  policy      = "${module.org_policy.policy_id}"
+  policy      = module.org_policy.policy_id
   name        = "terraform_members"
-  members  = "${var.members}"
+  members  = var.members
```

Comment:
Align spaces in code blocks

Reply:
Thank you, Pavlo
[Resolved](https://github.com/terraform-google-modules/terraform-google-vpc-service-controls/commit/63c5a8a98dbafd8436547a6d994293cafb99c825)